docs/architecture/modern-web-apps-azure/azure-hosting-recommendations-for-asp-net-web-apps.md

```diff
@@ -121,7 +121,7 @@ Your application's requirements should dictate its architecture. There are many
 
 Figure 11-2 shows an example reference architecture. This diagram describes a recommended architecture approach for a Sitecore content management system website optimized for marketing.
 
-![](./media/image11-2.png)
+![Figure 11-2](./media/image11-2.png)
 
 **Figure 11-1.** Sitecore marketing website reference architecture.
```

Comment:
Is the figure number mentioned here wrong ?

Reply:
Maybe, the previous paragraph does seem to describe the image https://github.com/dotnet/docs/blob/master/docs/architecture/modern-web-apps-azure/media/image11-2.png

Reply:
The figure numbering here needs to be adjusted in any case, since previous figures are not numbered. @ardalis, were you planning on adjusting the figure numbers? In the meantime, since there is only an explicit reference to Figure 1 in the next, I've changed the references to Figure 2 to refer to Figure 1, since only a Figure 1 is mentioned.